src/components/Rockets/Rocket.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { toggleReserve } from '../../redux/rockets/rockets';
+
+const Rocket = (props) => {
+  const {
+    rocketId, rocketName, rocketDescription, rocketImage, rocketReserved,
+  } = props;
+
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(toggleReserve({ id: rocketId }));
+  };
+
+  return (
+    <div className="rocket-container" key={rocketId}>
+      <div>
+        <img src={rocketImage} alt="/" />
+      </div>
+      <div className="content">
+        <h1>{rocketName}</h1>
+        <p>
+          {
+        rocketReserved
+          ? (
+            <span className="badge">Reserved</span>
+          ) : null
+}
+          {rocketDescription}
+        </p>
+        <div className="reserve-rocket">
+          {rocketReserved ? (<button type="button" onClick={handleClick} className="btn cancel">Cancel Reservation</button>) : (<button type="button" className="btn" onClick={handleClick}>Reserve Rocket</button>)}
```

Comment:
- [ ] Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
   - You can use the syntax:
   -  `{ rocketReserved && (  // code for reserved rockets )}`
   - `{ !rocketReserved && (  // code for not reserved rockets)}`

Reply:
Thank you for you review @thecodechaser implemented the changes